src/vst3/SurgeVst3Processor.cpp

```diff
@@ -507,15 +507,15 @@ tresult PLUGIN_API SurgeVst3Processor::process(ProcessData& data)
          float** outB = data.outputs[2].channelBuffers32;
          if (surgeInstance->activateExtraOutputs)
          {
-            for (int c = 0; c < 2; ++c)
+            for (int c = 0; c < 2; ++c) // c = channel, we have 2 (left, right)
             {
                outA[c][i] = (float)surgeInstance->sceneout[0][c][blockpos];
                outB[c][i] = (float)surgeInstance->sceneout[1][c][blockpos];
             }
          }
          else
          {
-            for( int c=0; c<2; ++c )
+            for (int c = 0; c < n_scenes; ++c)
```

Comment:
This one is also channel not scene. So need to fix this.

Reply:
Oh, ok! Easy. Should've assumed based on it being `c` not `s`!